client-side/test/loader.test.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
```

Comment:
please place the test file in the same hierarchy as in src folder,
for example this file related to loader, which located in src\stories, so its test should be in test\stories